model/src/w3sic4md.F90

```diff
@@ -514,6 +532,50 @@ SUBROUTINE W3SIC4 (A, DEPTH, CG, IX, IY, S, D)
            END DO
            WN_I= 0.5 * ALPHA
 
+#ifdef CESMCOUPLED
+        CASE (8) 
```

Comment:
@JessicaMeixner-NOAA - I'm happy to try to set up a call with CC,Lettie and @alperaltuntas  if that would help.
Please let me know.

Reply:
@lettie-roach - can you answer the above question from Jessica?

Reply:
We added case 8 as a new option for the attenuation scheme, so yes it's unrelated to the new cap. We have more changes to make to the attenuation scheme based on some recent results, so it would make sense to be a PR on its own